nsmd_probe_goals.go

```diff
@@ -0,0 +1,42 @@
+package main
```

Comment:
Please don't pull this machinery in... we need a much much much simpler NSMgr  that will not require this level of complexity for health check.